engine/src/main/java/io/camunda/zeebe/streamprocessor/ProcessingScheduleServiceImpl.java

```diff
@@ -68,18 +76,52 @@ public void runAtFixedRate(final Duration delay, final Task task) {
             }));
   }
 
-  private void scheduleOnActor(final Runnable task) {
+  private void useActorControl(final Runnable task) {
+    if (actorControl == null) {
+      LOG.debug("ProcessingScheduleService hasn't been opened yet, ignore scheduled task.");
+      return;
+    }
     actorControl.submit(task);
   }
 
+  public ActorFuture<Void> open(final ActorControl control) {
+    if (openFuture != null) {
+      return openFuture;
+    }
+
+    openFuture = new CompletableActorFuture<>();
+    writeRetryStrategy = new AbortableRetryStrategy(control);
+    final var writerFuture = writerAsyncSupplier.get();
+    control.runOnCompletionBlockingCurrentPhase(
```

Comment:
:question: Is it necessary to block the current phase? Sorry, it's not immediately obvious to me why this is required. What would happen if we didn't block it? The future may not be completed?

Reply:
Sorry this might be a leftover of the previous PR #10414 

Before I don't wanted to open/start the actor and fail it (so blocking the phase on actorStarting). Here the StreamProcessor started already anyway, and the actor is failed on error so we don't need it. Good catch thanks.